uint32 degree
---
bool success
